packages/compass-editor/src/types.ts

```diff
@@ -14,3 +14,16 @@ export type CompletionWithServerInfo = {
   /** Optional completion description */
   description?: string;
 };
+
+export type EditorRef = {
```

Comment:
Moved this here to be able to reference it from both `editor.tsx` and `actions-container.tsx` without introducing a circular import dependency.